tools/ci_build/github/azure-pipelines/templates/android-java-api-aar.yml

```diff
@@ -24,6 +24,11 @@ parameters:
   type: string
   default: ''
 
+- name: publish_executables
+  displayName: Publish executables such as onnxruntime_perf_test, ...
+  type: string
```

Comment:
there's also a boolean type.
if you want to use string, it may be useful to document what the expected values are (e.g., "0" and "1")

Reply:
I tried Boolean in some earlier commits in this PR, True and true are different, using string is more consistent, will add comments in a later PR